.github/workflows/ci-go.yml

```diff
@@ -41,10 +41,10 @@ jobs:
         run: pnpm install
 
       - name: Build & Unit Test
-        run: pnpm turbo run test --scope=cli
+        run: pnpm turbo -- run test --scope=cli --color
```

Comment:
Only the positional arguments were being passed in before:
```
./turbow.sh "run" "test"
```
This now gets sent as:
```
./turbow.sh "run" "test" "--scope=cli" "--color"
```

Likely a change in behavior when switching to pnpm from yarn